src/pal/src/loader/module.cpp

```diff
@@ -1620,6 +1620,8 @@ static HMODULE LOADLoadLibrary(LPCSTR shortAsciiName, BOOL fDynamic)
         shortAsciiName = FREEBSD_LIBC;
 #elif defined(__NetBSD__)
         shortAsciiName = "libc.so";
+#elif !HAVE_GNU_LIBNAMES_H
+        shortAsciiName = "libc.musl-x86_64.so";
```

Comment:
@rofl0r, on my box:

```
/lib/libc.musl-x86_64.so.1 -> /lib/ld-musl-x86_64.so.1
/usr/lib/libc.so -> /lib/ld-musl-x86_64.so.1
```

So `ld-musl-x86_64.so.1` is the actual binary and libc.so. Should we instead check `/lib/ld-musl-x86_64.so.1` ?